spec/factories/submission.rb

```diff
@@ -2,6 +2,7 @@
   factory :submission do
     description { Faker::Lorem.paragraph }
     state { :inactive }
+    deadline { Date.today }
```

Comment:
Is it worthwile making this a date further in the future to better represent what the real deadline would be?

Reply:
Agree!